src/transformers/dynamic_module_utils.py

```diff
@@ -143,9 +144,24 @@ def get_class_in_module(class_name, module_path):
     """
     Import a module on the cache directory for modules and extract a class from it.
     """
-    module_path = module_path.replace(os.path.sep, ".")
-    module = importlib.import_module(module_path)
-    return getattr(module, class_name)
+    with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
this is not the location to load the module. It's just to hold the file temporarily , and it will be copied back to the original place.